src/main/webapp/js/addrole.js

```diff
@@ -21,7 +21,7 @@ const addGlobalRole = () => {
         return;
     }
 
-    sendPostRequest(`${rootURL}/folder-auth/addGlobalRole`, response);
+    sendPostRequest(rootURL + "/folder-auth/addGlobalRole", response);
```

Comment:
Is there a reason for converting the template strings to concatenation using `+`?

Reply:
The template strings are unfortunately unsupported in the `WebClient` in JenkinsRule, which throws errors during testing. I've decided to remove template strings instead of using an alternative test suite.